src/app/games/abalone/AbaloneTutorial.ts

```diff
@@ -50,8 +43,27 @@ export class AbaloneTutorial extends Tutorial {
                 [_, _, O, _, _, N, N, N, N],
             ], 0),
             [AbaloneMove.ofSingleCoord(new Coord(4, 4), HexaDirection.RIGHT)],
-            $localize`Congratulations! You know everything you need to start a game!`,
+            TutorialStepMessage.CONGRATULATIONS(),
             TutorialStepMessage.FAILED_TRY_AGAIN(),
         ),
+        TutorialStep.fromPredicate(
+            TutorialStepMessage.TRANSLATIONS(),
+            $localize`To translate a group of piece, you must select the first piece, then select the last piece, then click on one of the 4 lateral direction the group can move without touching opponent pieces.<br/><br/>Go ahead, translate some pieces.`,
+            AbaloneRules.get().getInitialState(),
+            AbaloneMove.ofDoubleCoord(new Coord(2, 6), new Coord(4, 6), HexaDirection.UP),
+            (move: AbaloneMove, _ps: AbaloneState, _rs: AbaloneState) => {
+                if (move.isTranslation()) {
+                    return MGPValidation.SUCCESS;
+                } else {
+                    return MGPValidation.failure($localize`This is no translation, this is a "pushing move", try a translation.`);
+                }
+            },
+            $localize`Congratulations! You know everything you need to start a game!`,
+        ),
+        TutorialStep.informational(
+            TutorialStepMessage.RULES_CONFIGURATION(),
+            $localize`You will see when creating a game that you can configure two things.<br/><br/>First, the number of capture needed to win, this help create shorter game if wanted.<br/><br/>Last, the maximum size of a moved group. The default is 3 but if you want more chaotic and fun game, you could bump that limit up and play a "powerfull" alternative version of the game!`,
```

Comment:
> You will see when creating a game that you can configure two things.
> - The number of captureS needed to win. This WILL help SHORTENING the game length.
> - The maximum size of a moved group. The default is 3 but if you want A more chaotic and fun game, you could > INCREASE that limit and play a MORE POWERFUL alternative version of the game!

typos: (voir majuscules) number of capture -> number of captures, powerfull -> powerful, et quelques rephrasages minime ci-dessus

Tu donnes pas le défaut pour le premier paramètre, mais tu le donne pour le second. Donner le défaut pour les deux semble pas mal.

Sinon j'adhère à ce genre d'étape de tuto extra pour les jeux configurables !

Reply:
yep !